packages/jellyfish-api-core/src/category/loan.ts

```diff
@@ -131,6 +140,17 @@ export interface SetCollateralToken {
   activateAfterBlock?: number
 }
 
+export interface CollateralTokensData {
+  [key: string]: CollateralTokenDetail
```

Comment:
node side should probably change this to array

cc @surangap

Reply:
Diego is currently working on changing list* to return arrays